### PR TITLE
Helium GA: Runtime Manager Review

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Summary

Revising and proofing the Runtime Manager section for Helium GA.

Fix for cluster_telemetry_server_name is here: https://github.com/Kong/kong-ee/pull/3464
### Reason

[DOCU-2313](https://konghq.atlassian.net/browse/DOCU-2313)
### Testing

- while I was testing, I clicked on the link to [cluster_telemetry_server_name](cluster_telemetry_server_name)  on the [Runtime Config page](https://konnect--kongdocs.netlify.app/konnect/runtime-manager/runtime-instances/runtime-parameter-reference/) and it didn't go to the right section, nor was there a "cluster_telemetry_server_name" entry on the [Gateway Config Reference page](https://konnect--kongdocs.netlify.app/gateway/latest/reference/configuration/).

### CODE REVIEW

File: app/konnect/runtime-manager/runtime-groups/declarative-config.md [hunk 1]
```diff
@@ -17,9 +17,6 @@ separately.
 group.
 * Migrate configuration from one group to another.
 
-{:.note}
-> **Note:** To work with runtime groups, you need [decK v1.12.0 or later](/deck/).
```
Comment (lena-larionova) [inline app/konnect/runtime-manager/runtime-groups/declarative-config.md:21]: Q: why remove this note?

**Comment by cloudjumpercat** [comment]
> The update needs to be made here: https://github.com/Kong/kong-ee/blob/master/kong.conf.default and in the config reference docs that we have on the docs site (versions 2.6.x-2.8.x).
I suggest opening a DOCU ticket for this issue for tracking purposes, and you're welcome to take it yourself if you'd like.

@lena-larionova Should I just open a PR in the kong/kong-ee repo and add the description from [here](https://konnect--kongdocs.netlify.app/konnect/runtime-manager/runtime-instances/runtime-parameter-reference/)? Or do I also need to add additional information?

**Comment by lena-larionova** [comment]
> @lena-larionova Should I just open a PR in the kong/kong-ee repo and add the description from [here](https://konnect--kongdocs.netlify.app/konnect/runtime-manager/runtime-instances/runtime-parameter-reference/)? Or do I also need to add additional information?

Yes, go ahead and open a PR with that info and ping someone on the data team - they can validate if that's enough info. You can post the PR in Slack, in the `team-data` channel, to ask for review.

File: app/konnect/legacy/runtime-manager/renew-certificates.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ we recommend running the script again to create a new data plane with renewed
 certificates.
 
 1. Stop the data plane container.
-2. Open Runtime Manager and click **Configure New Runtime**.
+2. Open [Runtime Manager](https://konnect.konghq.com/runtime-manager/) and click **Configure New Runtime**.
```
Comment (lena-larionova) [inline app/konnect/legacy/runtime-manager/renew-certificates.md:31]:
```suggestion
2. Open [Runtime Manager](https://cloud.konghq.com/runtime-manager/) and click **Configure New Runtime**.
```

**Comment by lena-larionova** [comment]
Added a diagram from a deck Ross provided. There were a few of them, but this one seemed like the most useful.

File: app/konnect/runtime-manager/gateway-config.md [hunk 3]
```diff
@@ -3,21 +3,21 @@ title: Kong Gateway Configuration in Konnect
...
 
 Consumers, SNIs, upstreams, and certificates are all global. Plugins
 can either be global or scoped.
 
-### Gateway Services
+### Gateway services
 
 The **Gateway Service** configuration page lists all {{site.base_gateway}} services
-in the runtime group. Service entities are abstractions of each of your own
+in the runtime group. service entities are abstractions of each of your own
```
Comment (cloudjumpercat) [inline app/konnect/runtime-manager/gateway-config.md:20]:
```suggestion
in the runtime group. Service entities are abstractions of each of your own
```
This one is at the start of a sentence.

**Comment by cloudjumpercat** [comment]
@Guaris or @lena-larionova , can you re-approve? I think everything looks fine now. The one Vale error is for "Konnect" in a code comment, which I feel like Vale should skip over. Can we just ignore that now and add the rule to Vale later to ignore comments in the code? 

I also opened a PR for the cluster_telemetry_server_name issue, still waiting for the Kong Gateway team to approve and merge that. I'm guessing the updates to 2.6.x-2.8.x can be made in a separate PR from this one since those changes aren't related to Helium.

**Review by Guaris — APPROVED** [review]

**Pull request merged**